File: Ruby/01 - Introduction to Programming/Task04/optional_task.rb
#Task4 - optional_task.py
#------------------------------------------------------------#
#request input from user
print("Please input a number for num1: ")
num1 = gets.chomp()
print("Please input a second number for num2: ")
num2 = gets.chomp()

#print out variables before swapped values
puts "Before swap:"
puts("num1: " + num1)
puts("num2: " + num2)

#swap variable values by use of temp variable
temp = num1
num1 = num2
num2 = temp

#print out variables with swapped values
puts "After swap:"
puts("num1: " + num1)
puts("num2: " + num2)

File: Python/01 - Introduction to Programming/Task13/optional_task.py
#Task13 - optional_task.py
#------------------------------------------------------------#
#request input from user
num = int(input("Please enter a number less than 100: "))
while(num>=100):
	num = int(input("Please enter a number less than 100: "))

#determine if num is an odd or even number
if(num%2==0):
	print(str(num*2))
else:
	print(str(num*3))